plugins/USBqemu/qemu-usb/usb-ohci.cpp

```diff
@@ -356,7 +356,7 @@ static void ohci_copy_td(OHCIState *ohci, struct ohci_td *td,
 
     ptr = td->cbp;
     n = 0x1000 - (ptr & 0xfff);
-    if (n > len)
+    if (n > static_cast<uint32_t>(len))
```

Comment:
IMHO, it is better to change the function len type to size_t. And to drop the horrible cast.